docs/docs/getting-started/04-examples/02-editor-example.md

```diff
@@ -15,88 +15,94 @@ on the page to expand either the code editor or graph view to full screen.
+            component: timer
+            transition: configure
+        on_configure:
+          lifecycle:
+            component: timer
+            transition: activate
 ```
 
-Disabling one or both auto event switches will also remove the corresponding events from the YAML, while enabling the
-switch will regenerate the event in the YAML.
+Disabling one or both auto-lifecycle event switches will also remove the corresponding events from the YAML, while
+enabling the switch will regenerate the event in the YAML.
 
-<!-- TODO: Replace the following sections with adding interactive buttons 
+## Setting component parameters
 
-Show how we can:
-- go to the sidebar
-- click on add trigger button
-- drag to reposition
-- create an event edge to the timer
-- choose the event type (e.g. deactivate)
-- rename the button
+Component parameters can be edited directly in the settings. Try changing the value of the Timeout parameter to a
+different one and look for the corresponding change in the code editor. Then press play on the application and observe
+that the timer now has a different duration before triggering the "Is timed out" predicate.
 
-Repeat for a second button to show manual deactivation / activation
+## Adding and deleting elements
 
-Then say: now that we learned to add buttons and create event edges, try to do the same for components. As an exercise,
-search for and add timer components, set thea auto-lifecycle events, and connect the transition edges to recreate the
-example from scratch.
--->
+Let's now make the example a bit more interesting by adding more elements to the graph using the sidebar menu; press the
+(+) button in the corner of the graph to open the sidebar. At the top, you will see special entries
+"Hardware Interface", "Trigger Events Button", "Sequence" and "Condition", for which a dedicated example is included
+in later sections. These are followed by a list of components with a name and brief description.
+
+Trigger Events Buttons are interactive elements that can be used to trigger events through mouse clicks and interact
+with the flow of the application. Click on the Trigger Events Button in the sidebar to add such an element to the
+application. A new button should appear on the graph and in the code.
+
+The sidebar menu can be closed by clicking anywhere in the graph. Drag on the button to change its position in the
+graph. This will also update the corresponding Trigger Events Button position in the code editor.
+
+To delete an element from the graph, press the small menu icon in its top right corner, then click Remove.
 
 ## Creating and deleting event edges
 
-The timers are connected with a Transition event edge. To change the event type on an edge, click on the label. This
-will open a selection menu showing other available event types (for example, Load, Unload, Configure...). Choosing
+The timers are already connected with a Transition event edge. To change the event type on an edge, click on the label.
```

Comment:
The sentence starting with "to change the event type on an edge..." is confusing to me, not sure if it is the usage of the word label or maybe the order of sentences between this and the next one where I feel its not quite clear why/ what we can change to.

Reply:
hmm, feels quite clear to me. Not sure how I could improve it

Reply:
[Screencast from 2024-12-19 11-18-40.webm](https://github.com/user-attachments/assets/ff8781d8-07ae-4a5f-8eb4-27c0047dd60a)

Talking about this, do you have a suggestion?

Reply:
Sure, my point was moreso making it clear why the user would want to change the edge from transition to something else -- but maybe they already have that knowledge from elsewhere? @domire8